src/components/Button/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import { ButtonVariant } from './types';
+
+import type { IButton } from './types';
+
+import './Button.scss';
+
+const Button = ({
+  children,
+  variant = ButtonVariant.Outline,
+  type = 'button',
+  size,
+  className,
+  ...props
+}: IButton) => {
+  const buttonClass = {
+    [ButtonVariant.Outline]: 'outline',
+    [ButtonVariant.Contained]: 'contained',
+    [ButtonVariant.Default]: '',
+  };
+
+  return (
+    <button
+      {...props}
+      className={`button button-${buttonClass[variant]} ${size ? 'large' : ''} ${className || ''}`}
```

Comment:
you need to change size from boolean value to type `large`, `small`, `medium` for example